test/jdk/jdk/jfr/event/profiling/TestMultipleRecordings.java

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
What is the purpose of this test?

Reply:
Tests that restarting the sampler works.

Reply:
It should be named something related to the CPU sampler, then. We already have many tests for starting multiple recordings.

Reply:
I removed it, as I split the test in two and `TestCPUTimeMultipleRecordings.java` therefore already exists.